ocaml/database/jbuild

```diff
@@ -71,4 +72,41 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
   ))
  )
 )
+
+(executable
+ ((name unit_test_marshall)
+  (public_name unit_test_marshall)
+  (package xapi)
+  (modules (
+    unit_test_marshall
+    ))
+  (libraries (
+    xapi-database
+  ))
+ )
+)
+
+(executable
+ ((name database_server_main)
+  (public_name database_server)
```

Comment:
If you set a public name it'll get installed, is that what you intended?

Reply:
Ah yes, copy-pasta mistake - thanks!